lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -108,6 +117,17 @@ def setup(self):
                     " use --overwrite to overwrite"
                 )
 
+        if not (self.source_ra or self.source_dec):
+            self.source_pos = SkyCoord.from_name(self.source_name)
+        elif bool(self.source_ra) != bool(self.source_dec):
+            raise ToolConfigurationError(
+                "Either provide both RA and DEC values for the source or none"
+            )
+        else:
+            self.source_pos = SkyCoord(
+                ra=self.source_ra, dec=self.source_dec, unit="deg"
```

Comment:
Do not give the unit here. Since `source_ra` is a string, that should contain the unit.